doc/product-variants-migration.md

```diff
@@ -0,0 +1,24 @@
+# Product variants migration
+
+If you used variant groups or inner variation types (with a paid extension for the Enterprise Edition), the product migration is done in two steps. At first the products are migrated as they are not taking variants in account. Then in a second step, family variants, product models and product variants are created in the PIM 2.0 to reproduce the variants of the PIM 1.7.
```

Comment:
Maybe it would be could to give the name of the extension: Inner Variation Bundle

Reply:
Maybe we should be more clear about the three cases we handle: variant groups, IVB and IVB + variant groups